tools/cli/src/mentat_cli/repl.rs

```diff
@@ -76,17 +66,27 @@ impl Repl {
     pub fn run(&mut self, startup_commands: Option<Vec<Command>>) {
         let mut input = InputReader::new();
 
+        let mut unexecuted_commands: Vec<InputResult> = Vec::new();
+
         if let Some(cmds) = startup_commands {
-            for command in cmds.iter() {
-                println!("{}", command.output());
-                self.handle_command(command.clone());
-            }
+            unexecuted_commands.append(&mut cmds.into_iter().map(|c| MetaCommand(c.clone())).collect());
```

Comment:
`.append(iter.collect())` is equivalent to `.extend(iter)`, I think.  Also, `into_iter()` takes ownership of each of the elements, so you don't need the inner `clone`: just
```rust
unexecuted_commands.extend(cmds.into_iter().map(MetaCommand))
```
should do it, I think.  (Please correct me if I'm wrong: I'll learn something!)